package.json

```diff
@@ -5,32 +5,33 @@
   "main": "index.js",
   "repository": {
     "type": "git",
-    "url": "git@bitbucket.org:rulemotion/resin-jwt"
+    "url": "git@github.com:resin-io-modules/resin-jwt"
   },
   "author": "Aleksis Brezas <abresas@resin.io>",
   "scripts": {
     "lint": "resin-lint ./index.coffee ./test",
     "pretest": "npm run lint && coffee -c index.coffee",
-    "test": "mocha --compilers coffee:coffee-script/register test/index.coffee",
+    "test": "mocha --require coffeescript/register test/index.coffee",
     "prepublish": "npm test"
   },
   "dependencies": {
     "bluebird": "^3.0.0",
-    "jsonwebtoken": "^5.0.4",
-    "passport-jwt": "^2.1.0",
+    "jsonwebtoken": "^8.3.0",
```

Comment:
`createJwt` directly passes its args through to jsonwebtoken, can you check that none of the args it passes are affected by the breaking changes in jsonwebtoken

Reply:
Yeah, there don't appear to be any changes that affect us.